api/cluster-resources/build.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
```

Comment:
Nice example how an isolated bit of build logic can be separate from devstart but still well integrated with the build!